blob/blob.go

```diff
@@ -6,54 +6,162 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/tendermint/tendermint/crypto/merkle"
```

Comment:
Looks like it should be `"github.com/celestiaorg/go-square/merkle"` 😉

Reply:
Same for other files.

Reply:
Hm, I see that in `go.mod` `github.com/celestiaorg/go-square/merkle` got removed. Also, in `go-square/v2` there is no `merkle` package anymore. So, I might be wrong about bad import.